src/sentry/grouping/strategies/configurations.py

```diff
@@ -178,6 +177,15 @@ def register_strategy_config(id: str, **kwargs) -> type[StrategyConfiguration]:
           no longer result in different issues.
     """,
     initial_context={
+        # These come from newstyle:2019-05-08
```

Comment:
https://github.com/getsentry/sentry/blob/3e9f33e84adf10805241a0c260056bb5780f05c7/src/sentry/grouping/strategies/configurations.py#L135-L141

`"with_exception_value_fallback": True,` is set to `False` in line 187 since `newstyle:2019-10-29` overwrites the value.